drivers/display/ssd1322.c

```diff
@@ -28,10 +29,20 @@ struct ssd1322_config {
 	uint16_t height;
 	uint16_t width;
 	uint16_t column_offset;
+	uint8_t row_offset;
+	uint8_t start_line;
+	uint8_t mux_ratio;
+	bool remap_row_first;
+	bool remap_columns;
+	bool remap_rows;
+	bool remap_nibble;
+	bool remap_com_odd_even_split;
+	bool remap_com_dual;
+	uint8_t segments_per_pixel;
 };
 
-static inline int ssd1322_write_command(const struct device *dev, uint8_t cmd,
-					const uint8_t *buf, size_t len)
+static inline int ssd1322_write_command(const struct device *dev, uint8_t cmd, const uint8_t *buf,
```

Comment:
Please do not make formatting changes like this in the same commit as functional additions. If you would like to format a file before editing, IE with clang-format, then please do so and keep the formatting changes in *one commit*. This makes the review process much easier